spec/integration/post_show_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'rails_helper'
+
+RSpec.describe 'post#show', type: :system do
+  let!(:user_one) { User.create(name: 'Oscar', photo: 'https://robohash.org/1', bio: 'bio', post_counter: 0) }
+  let!(:user_two) { User.create(name: 'Carlos', photo: 'https://robohash.org/1', bio: 'bio', post_counter: 0) }
+  let!(:post) do
+    Post.create author_id: user_two.id, comments_counter: 0, likes_counter: 0, title: 'title', text: 'something'
+  end
+  let!(:comment_one) { Comment.create author: user_one, post:, text: 'hello world' }
+  let!(:comment_two) { Comment.create author: user_two, post:, text: 'hello rails' }
+
+  describe 'Rendering' do
+    it 'Should show the post title' do
+      visit "users/#{user_two.id}/posts/#{post.id}"
+
+      expect(page).to have_content(user_two.name)
+    end
+
+    it 'Should show the post comments counter' do
+      visit "users/#{user_two.id}/posts/#{post.id}"
+
+      expect(page).to have_content("comments: #{post.comments_counter}")
+    end
+
+    it 'Should show the post likes counter' do
+      visit "users/#{user_two.id}/posts/#{post.id}"
+
+      expect(page).to have_content("likes: #{post.likes_counter}")
+    end
+
+    it 'Should show the post body' do
```

Comment:
Hi @thecodechaser 
Thank you for your review!
I have a question for you on this point:
- Isn't this test requirement covered by this spec?

![image](https://github.com/luis-pomare/rails-blog-app/assets/80157011/dfabc41c-132d-4d1b-af24-fe3bff0321cc)

I going to change the ´it´ description and put "author of the posts" instead of user name. is that enough?